mkosi/distributions/serpentos.py

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: LGPL-2.1+
+
+import shutil
+from collections.abc import Sequence
+
+from mkosi.config import Architecture
+from mkosi.context import Context
+from mkosi.distributions import Distribution, DistributionInstaller, PackageType
+from mkosi.installer.moss import invoke_moss
+from mkosi.log import die
+
+
+class Installer(DistributionInstaller):
+    @classmethod
+    def pretty_name(cls) -> str:
+        return "Serpent OS"
+
+    @classmethod
+    def filesystem(cls) -> str:
+        return "btrfs"
+
+    @classmethod
+    def package_type(cls) -> PackageType:
+        return PackageType.moss
+
+    @classmethod
+    def default_release(cls) -> str:
+        return "rolling"
+
+    @classmethod
+    def default_tools_tree_distribution(cls) -> Distribution:
+        return Distribution.serpentos
+
+    @classmethod
+    def tools_tree_packages(cls) -> list[str]:
+        return [
+            "dash", # for sh
+            "systemd",
+            "zstd",
+        ]
+
+    @classmethod
+    def setup(cls, context: Context) -> None:
+        if not shutil.which("moss"):
+            die(f"moss binary not found but needed to install {cls.pretty_name()}")
```

Comment:
No need for this since we don't do it for the other distributions either. We'll add proper checks at some point but we'll do it for all distributions at once